core/db/models.py

```diff
@@ -32,10 +34,27 @@ class Meta:
 class ConnectionPeerModel(BaseModel):
     user = ForeignKeyField(UserModel, backref="peer", on_delete="CASCADE")
     public_key = CharField()
+    private_key = CharField()
     preshared_key = CharField()
     shared_ips = CharField()
     peer_name = CharField(default=None, null=True)
 
+    def setup_peer_keys(self):
+        peer_private_key = private_key()
+        self.privatekey = peer_private_key
+        self.public_key = public_key(peer_private_key)
+        self.preshared_key = preshared_key()
+
+    def peer_for_wg_server_config(self):
+        return f"""
+#{self.peer_name}
+[Peer]
+PublicKey = {self.public_key}
+PresharedKey = {self.preshared_key}
+AllowedIPs = {self.shared_ips}/32
+
+"""
```

Comment:
same as above.

Reply:
There are 3 reasons to leave it here
1) Peer name can be changed, so it's better to change them into the Wireguard server config too, to make searching for peers in wghub easier
2) It'll be easier to use it from the model entity
3) If we'll add it to db_works, it'll stay in the database. That will create an ugly database entity that was made from another part of it's row

Reply:
then move it to `core.db.model_serializer.ConnectionPeer`. this method does not work with the model, so it has nothing to do here. as a bonus, using such a method in a serialized model ensures that the data will be correct.